### PR TITLE
Added new features Double-hop, Traffic Optimizer and DNS Monitoring in UI

### PR DESCRIPTION
**Pull request by renishnair-viper**
As VPNSDKv2 has been updated we have included new features Double-hop, Traffic Optimizer, and DNS Monitoring in the UI.
VpnSDK, OpenVPN.Binaries.Scramble, OpenVPN.Certificates.WLVPN and VpnSDK.Private.OpenVpn has been updated. 
Update OpenVPN.Binaries.Scramble nuGet package to version 2.5.6.
Updated VpnSDK nuGet package to version 2.2.40.
Updated OpenVPN.Certificates.WLVPN package to 1.1.1

### CODE REVIEW

File: src/App/Properties/Strings.en.resx [hunk 1]
```diff
@@ -631,4 +631,28 @@ The new version is {1} (You currently have version {2})</value>
...
+    <value>Traffic Optimizer</value>
+  </data>
+  <data name="EnableTrafficOptimizer" xml:space="preserve">
+    <value>Enable traffic optimizer</value>
+  </data>
+  <data name="DoubleHop" xml:space="preserve">
+    <value>Double Hop</value>
+  </data>
+  <data name="MaxThreeApplication" xml:space="preserve">
+    <value>Please note that a maximum of 3 applications can only be optimized.</value>
```
Comment (rastapopoulos1) [inline src/App/Properties/Strings.en.resx]: Please change it to

`Please note that only up to 3 applications can be optimized at a time.`

File: src/App/App.csproj [hunk 2]
```diff
@@ -955,7 +960,10 @@
       <Version>1.3.7</Version>
     </PackageReference>
     <PackageReference Include="VpnSDK">
-      <Version>2.2.32</Version>
+      <Version>2.2.33</Version>
+    </PackageReference>
+    <PackageReference Include="VpnSDK.Private.OpenVpn">
+      <Version>1.0.37-develop00010</Version>
```
Comment (rastapopoulos1) [inline src/App/App.csproj]: Please change it to refer to 1.0.38 version instead

Comment (renishnair-viper) [reply src/App/App.csproj]: Resolved

File: src/App/Properties/Strings.en.resx (hunk 1 above)
Comment (renishnair-viper) [reply src/App/Properties/Strings.en.resx]: Resolved

**Review by rastapopoulos1 — APPROVED** [review]

**Review by vimalraj-adikesavan — APPROVED** [review]

**Pull request merged**